middleware.ts

```
@@ -6,7 +6,7 @@ export async function middleware(request: NextRequest) {
   const { pathname } = request.nextUrl;
 
   // Rutas públicas
-  if (pathname === "/contacto" || /^\/producto\/\d+(?:\/|$)/.test(pathname)) {
+  if (pathname === "/contacto" || /^\/producto\/\d+(?:\/|$)/.test(pathname) || "/cart") {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)